src/foraging_gui/Foraging.py

```diff
@@ -4242,25 +4242,23 @@ def _StartTrialLoop(self,GeneratedTrials,worker1,worker_save):
                 # calculate bias every 10 trials
                 if (GeneratedTrials.B_CurrentTrialN+1) % 10 == 0 and GeneratedTrials.B_CurrentTrialN+1 > 20:
                     # correctly format data for bias indicator
-                    choice_history = [1 if x == 2 else int(x) for x in self.GeneratedTrials.B_AnimalResponseHistory]
-                    any_reward = [any(x) for x in np.column_stack(self.GeneratedTrials.B_RewardedHistory)]
-
-                    # use self.bias_n_size of trials to compute bias over or take n the first 0 to N trials
-                    l = len(choice_history)
-                    # FIXME: .6  of choice history is a little arbitrary. If the n_trial_back is close to equaling the
-                    #  trial count, the logistic regression can't be calculated because of an error saying
-                    #  'Cannot have number of splits n_splits=10 greater than the number of samples: 2'
-                    n_trial_back = self.bias_n_size if l > self.bias_n_size else \
-                        round(len(np.array(choice_history)[~np.isnan(choice_history)])*.6)
+
+                    formatted_history = [np.nan if x == 2 else int(x) for x in self.GeneratedTrials.B_AnimalResponseHistory]
+                    formatted_reward = [any(x) for x in np.column_stack(self.GeneratedTrials.B_RewardedHistory)]
+
+                    # only take last 200 trials if enough trials have happened
+                    choice_history = formatted_history[-200:] if len(formatted_history) > 200 else formatted_history
+                    any_reward = formatted_reward[-200:] if len(formatted_reward) > 200 else formatted_reward
+
                     # add data to bias_indicator
                     if not self.bias_thread.is_alive():
                         logger.debug('Starting bias thread.')
                         self.bias_thread = threading.Thread(target=self.bias_indicator.calculate_bias,
-                                                       kwargs={'time_point': self.GeneratedTrials.B_TrialStartTime[-1],
+                                                       kwargs={'trial_num': len(formatted_history),
                                                                'choice_history': choice_history,
                                                                'reward_history': np.array(any_reward).astype(int),
-                                                               'n_trial_back': n_trial_back,
-                                                               'cv': 2})
+                                                               'n_trial_back': 5,
+                                                               'cv': 5})
```

Comment:
we should use cv = 1

Reply:
When I use cv=1 I get this error: 
`sklearn.utils._param_validation.InvalidParameterError: The 'cv' parameter of LogisticRegressionCV must be an int in the range [2, inf), an object implementing 'split' and 'get_n_splits', an iterable or None. Got 1 instead.`

I thought on Friday we talked about changing it to 5, but maybe there was a conversation that I missed about something different

Reply:
Are you sure you are on the v0.11.2?